docs/data/product/dea-fractional-cover-landsat/_data.yaml

```diff
@@ -6,7 +6,7 @@
 title: DEA Fractional Cover (Landsat)
 long_title: Geoscience Australia Landsat Fractional Cover Collection 3
 header_image: /_files/cmi/FractionalCover1.jpg
-version: 2.3.1
+version: "Landsat 5/7: 2.5.0, Landsat 8/9: 2.5.1"
```

Comment:
Does this mean data from Landsat 5/7 and Landsat 8/9 will be located in different subfolders on S3?

Reply:
![ksnip_20240717-100507](https://github.com/user-attachments/assets/c5729bac-19ef-4d7d-9535-b58369455125)

Good question - to avoid confusion, I've now added some explanatory text.

Reply:
> Does this mean data from Landsat 5/7 and Landsat 8/9 will be located in different subfolders on S3?

yes, they are.